Bootstrap Docker and Add README.md

shmashing opened the pull request:
This commit bootstraps the docker builds for the service and adds the README that tells how to do allllll the things

shmashing committed:
Bootstrap Docker and Add README.md

This commit bootstraps the docker builds for the service and adds the README that tells how to do allllll the things

shmashing commented:
@jaredonline ready for your 👀

shmashing committed:
Fix Typo in README

shmashing committed:
Added Clarifying Info

This commit adds a bit of clarifying information to the README and fixes another copy/paste error :D

jaredonline commented on README.md:
```diff
@@ -0,0 +1,61 @@
+# Destiny Scrims Tracker #
+Version 1.0.0
```
No way we're at `1.0.0` yet hombre! More like `0.0.1`

shmashing replied:
waaat

jaredonline replied:
1.0.0 implies a solid interface and functionality. 0.0.1 implies things are changing rapidly, don't rely on anything.

shmashing replied:
😂 na i know just forgot to change it

jaredonline commented on README.md:
```diff
@@ -0,0 +1,61 @@
+
+- [Guardian Object](Destiny.ScrimsTracker.Logic/Models/Guardian.cs)
+- [GuardianMatchResult Object](Destiny.ScrimsTracker.Logic/Models/GuardianMatchResult.cs)
+- [Match Object](Destiny.ScrimsTracker.Logic/Models/Match.cs)
```
These don't actually exist yet, right?

shmashing replied:
They do!

jaredonline replied:
Ah, just found them. Can they get deleted and re-added via a PR. I have thoughts.

shmashing replied:
Mostly they just there for placeholders. Or did you have thoughts on where they should be?

jaredonline replied:
No, on their contents and which models we should have. I have no idea where things should be in a .NET project

shmashing replied:
Done

jaredonline commented on README.md:
```diff
@@ -0,0 +1,61 @@
+
+To build the service locally, from the root of the repository, run `dotnet build`.
+
+To run the service locally, from the root of the repository, run 'dotnet run --project Destiny.ScrimsTracker.Api/Destiny.ScrimsTracker.Api.csproj'
```
```suggestion
To run the service locally, from the root of the repository, run `dotnet run --project Destiny.ScrimsTracker.Api/Destiny.ScrimsTracker.Api.cspro`'
```

jaredonline commented on README.md:
```diff
@@ -0,0 +1,61 @@
+
+To run the migration, execute the console app with the following command:
+`dotnet run -s <server endpoint> -d <database name> -p <port> -u <user id> -w <password>`
+
```
Can me Makify these commands as:

- `make build`
- `make build_ds`
- `make run`
- `make run_ds`
- `make test`
- `make run_migrations` - this one might be hard with the need for the password and ish

It's fine if this happens in a later PR

shmashing replied:
yea! I'm unfamiliar with all that tho

jaredonline replied:
I can do it once this is merged

shmashing committed:
Update README.md

Co-Authored-By: Jared McFarland <jaredonline@users.noreply.github.com>

shmashing committed:
Update README.md

jaredonline commented on README.md:
```diff
@@ -0,0 +1,60 @@
+
+To build the service locally, from the root of the repository, run `dotnet build`.
+
+To run the service locally, from the root of the repository, run `dotnet run --project Destiny.ScrimsTracker.Api/Destiny.ScrimsTracker.Api.cspro`'
```
```suggestion
To run the service locally, from the root of the repository, run `dotnet run --project Destiny.ScrimsTracker.Api/Destiny.ScrimsTracker.Api.cspro`
```

shmashing committed:
Remove My Beautiful Models

This commit removes the placeholder models i put in. They will not die in vain.

shmashing committed:
Merge branch 'bootstrap-service' of github.com:shmashing/destiny-scrim-tracker into bootstrap-service

shmashing committed:
Update README.md

Co-Authored-By: Jared McFarland <jaredonline@users.noreply.github.com>

shmashing committed:
Update README.md
